src/libstore/ssh-store.cc

```diff
@@ -82,17 +85,73 @@ class SSHStore : public virtual SSHStoreConfig, public virtual RemoteStore
     };
 };
 
+/**
+ * The mounted ssh store assumes that filesystems on the remote host are shared
+ * with the local host. This means that the remote nix store is available
+ * locally and is therefore treated as a local filesystem store.
+ */
+class MountedSSHStore : public virtual SSHStore, public virtual LocalFSStore
+{
+public:
+
+    MountedSSHStore(const std::string & scheme, const std::string & host, const Params & params)
+        : StoreConfig(params)
+        , RemoteStoreConfig(params)
+        , SSHStoreConfig(params)
+        , Store(params)
+        , RemoteStore(params)
+        , SSHStore(scheme, host, params)
+        , LocalFSStoreConfig(params)
+        , LocalFSStore(params)
+    {
+        extraRemoteProgramArgs = "--process-ops --allow-perm-roots";
+    }
+
+    static std::set<std::string> uriSchemes()
+    {
+        return {"mounted-ssh"};
```

Comment:
Should this be `mounted-ssh-ng`?

Reply:
Yeah good idea. I wish we had already deprecated the other one...but we haven't.

Reply:
Fixed now. 👍